### PR TITLE
Wikigeo full page

### PR DESCRIPTION
**Pull request by tzufgoogle**
Wikigeo data where instead of taking the 3 sentences in a page as one sample takes all sentences and creates a sample per sentence

### CODE REVIEW

**Review by sayalikulkarni — CHANGES_REQUESTED** [review]
initial inputs - based on Pittsburgh data. 
also related to the data - 
$ wc -l /mnt/hackney/data/cabby/wikigeo/wikigeo_pittsburgh_raw_7.json
23209 /mnt/hackney/data/cabby/wikigeo/wikigeo_pittsburgh_raw_7.json

$ wc -l  /mnt/hackney/data/cabby/wikigeo/v11/pittsburgh_*.json
   1991 /mnt/hackney/data/cabby/wikigeo/v11/pittsburgh_dev.json
   1991 /mnt/hackney/data/cabby/wikigeo/v11/pittsburgh_test.json
  15926 /mnt/hackney/data/cabby/wikigeo/v11/pittsburgh_train.json
  19908 total

v11 should have much more examples than the older ones like v7 - no? Whats missing?

File: cabby/data/extract_test.py [hunk 1]
```diff
@@ -14,6 +14,12 @@
 
 '''Tests for wikidata.py'''
 
+
+
+import sys
+sys.path.append("/home/tzuf_google_com/dev/cabby")
```
[the remark this answers is not in the document]
Comment (tzufgoogle) [reply cabby/data/extract_test.py]: removed

File: cabby/data/wikipedia/query.py [hunk 2]
```diff
@@ -83,11 +99,11 @@ def clean_text(text: Text) -> Text:
     The text after parsing.
   '''
   # Remove titles.
-  clean_text = text.split('== See also ==')[0]
-  clean_text = clean_text.split('== Notes ==')[0]
-  clean_text = clean_text.split('== References ==')[0]
-  clean_text = re.sub(r'=.*=', r'.', clean_text)
-  return clean_text
+  text = text.split('== See also ==')[0]
```
[the remark this answers is not in the document]
Comment (tzufgoogle) [reply cabby/data/wikipedia/query.py:102]: Added External links removal

**Review by tzufgoogle — COMMENTED** [review]
Build, test run working

**Comment by tzufgoogle** [comment]
The comparison sould be to - /mnt/hackney/data/cabby/wikigeo/wikigeo_pittsburgh_raw_9.json
(this version doesn't have duplicate text).  This version has 18,258  wherase the current v11 version has 19,908.

**Comment by tzufgoogle** [comment]
Fixed now there are 33216 for Pittsburgh

**Review by sayalikulkarni — APPROVED** [review]
good to go...

**Pull request merged**